sftp/stow_test.go

```diff
@@ -0,0 +1,163 @@
+package sftp
+
+import (
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/graymeta/stow"
+	"github.com/graymeta/stow/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStow(t *testing.T) {
+	config := stow.ConfigMap{
```

Comment:
Is there a reason the private key file and passphrase aren't checked here? It doesn't look like the test allows for an SFTP_PASSWORD value.

WDYT about assembling a list of missing env vars to put in the skip message?

Reply:
yeah, i don't run any ssh servers that allow password auth, so couldn't test against them. The tests are only set up to allow key based access.